third_party/txt/BUILD.gn

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+declare_args() {
+  flutter_use_fontconfig = false
+}
+
```

Comment:
Assert that you are on linux here as it is the only platform on which FC is supported.